src_feedback/elf.rs

```diff
@@ -0,0 +1,666 @@
+use std::mem;
+use std::fs::{File};
+use std::io::{Error, ErrorKind, Read, Seek, SeekFrom};
+use std::borrow::BorrowMut;
+
+use std::cell::RefCell;
+
+use crate::tools::{search_address_opt_key, extract_string};
+
+const EI_NIDENT: usize = 16;
+
+#[allow(non_camel_case_types)]
+type Elf64_Addr = u64;
```

Comment:
I think the libc crate defines these already (https://docs.rs/libc/0.2.54/libc/type.Elf64_Addr.html) as well as [Elf64_Ehdr](https://docs.rs/libc/0.2.54/libc/struct.Elf64_Ehdr.html). Does it make sense to use the libc:: defintions of those?

Reply:
Yes, sure!  We should replace use it.